src/runner.ts

```diff
@@ -47,6 +47,11 @@ const processOptions = async (options: RunnerOptions) => {
       );
     }
 
+    // Prevent node to emit an uncaught error and exit when the backend restarts or goes down
+    pgPool.on('error', err => {
+      console.error('The PostgreSQL client emitted an error: ', err.message);
+    });
+
```

Comment:
Thanks for spotting my lack of this; please can you copy the code from here as it has a comment explaining it too:

https://github.com/graphile/postgraphile/blob/d2374174adfb363c20fc068aea255ad45bbbe708/src/postgraphile/postgraphile.ts#L211-L222

Thanks :pray: